examples/mir_demo_server/server_example.cpp

```diff
@@ -116,6 +119,38 @@ try
 {
     miral::MirRunner runner{argc, argv, "mir/mir_demo_server.config"};
 
+    miral::InputConfiguration input_configuration;
+
+    miral::ConfigFile test{runner, "mir_demo_server.input", miral::ConfigFile::Mode::reload_on_change,
+        [&input_configuration](auto& in, auto path)
+    {
+        std::cout << "** Reloading: " << path << std::endl;
```

Comment:
Should we use `mir::log_info(...)` in the example instead?

Reply:
There's a lot more `cout` than anything else in the examples: No `log_XXX` one instance of `mir::log()`.

So this is consistent with the example code. But maybe we want to change?

Reply:
I'm okay with `cout` if it is consistent. I didn't know whether or not we're pushing compositor authors to use mir's logging facilities (I am using them in miracle, for example)

Reply:
Maybe we should be. (I also do it in Miriway and Frame)

But that's another PR...